frontend/src/container/Account.jsx

```diff
@@ -313,10 +314,7 @@ export class Account extends React.Component {
   setHeadTitle = () => {
     const { props } = this
     if (props.system.config.instance_name) {
-      GLOBAL_dispatchEvent({
-        type: CUSTOM_EVENT.SET_HEAD_TITLE,
-        data: { title: buildHeadTitle([props.t('My Account'), props.system.config.instance_name]) }
-      })
+      props.dispatch(setHeadTitle(buildHeadTitle([props.t('My Account'), props.system.config.instance_name])))
```

Comment:
```suggestion
      props.dispatch(setHeadTitle(buildHeadTitle([props.t('My Account'), props.system.config.instance_name])))
```
I think this kind of code should be factored into a function in tracim_frontend_lib, since it is everywhere in the frontend code. Not sure this should block this PR, but if you are so kind as to free us of this, I'd be very grateful.

(cc @grignards what do you think?)